### PR TITLE
Update to use factory in hurumap-ui

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

Requires: https://github.com/CodeForAfrica/HURUmap-UI/pull/62

Update to use factory in hurumap-ui

## Type of change

Please delete options that are not relevant.

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by karimkawambwa** [comment]
@kilemensi @KhadijaMahanga kindly review.

Using the new hurumap version.

File: src/data/useChartDefinitions.js [hunk 1]
```diff
@@ -1,20 +1,5 @@
...
-        chart.queryAlias = `chart${index}`;
-        const { visual, stat } = chart;
-        visual.queryAlias = `viz${index}`;
-        stat.queryAlias = `viz${index}`;
-        index += 1;
-      })
-    );
-    return sectionedCharts;
-  }, []);
+export default () => useMemo(() => assignQueryAlias(sectionedCharts), []);
```
Comment (kilemensi) [inline src/data/useChartDefinitions.js]: Thinking out loud: Couldn't we have made `useMemo` part of `assignQueryAlias` or something? Unless this behaviour changes on a per app basis.

File: src/data/charts.json [hunk 2]
```diff
@@ -855,6 +957,9 @@
       {
         "title": "Cereal yield in kg per hectare (Thousands)",
         "visual": {
+          "props": {
+            "horizontal": true
```
Comment (KhadijaMahanga) [inline src/data/charts.json]: What happened to setting `horizontal` depending on data length?

File: src/config.js [hunk 3]
```diff
@@ -127,7 +127,20 @@ const config = {
...
+    'allPopulationSex2007S',
+    'allPopulationSex2009S',
+    'allPopulationSex2011S',
+    'allPopulationSex2012S',
+    'allPopulationSex2013S',
+    'allPopulationSexYears',
+    'allPopulationResidence2009S',
+    'allPopulationResidence2012S',
+    'allPopulationResidence2013S',
+    'allTotalPopulations'
```
Comment (KhadijaMahanga) [inline src/config.js]: :+1:  nice move!

File: src/data/charts.json (hunk 2 above)
Comment (karimkawambwa) [reply src/data/charts.json]: @kilemensi is still a thing, for now this has to be manually defined. 

Using ChartFactory from Dominion.

**Comment by karimkawambwa** [comment]
@KhadijaMahanga @kilemensi resolved the chart issues https://github.com/CodeForAfrica/HURUmap-UI/pull/65

Perhaps once reviewed there we can complete this or if possible try test the package before publishing with `yarn publish:local`.

**Comment by karimkawambwa** [comment]
> 🚀
> 
> On clicking container action (share, dowload), the grid position shifts.
> Screenshot from 2019-10-17 18-05-47

@KhadijaMahanga this will be addressed in a separate InsightChartcontainer improvements.

**Comment by KhadijaMahanga** [comment]
>  @karimkawambwa
> 
> Was debugging what was causing this space issue:
> Screenshot from 2019-10-18 15-38-39
> 
> The way we're accessing `variant` in makeStyles is wrong in HURUmap UI & hence it was evaluated to `undefined`. I'll fix them there:https://github.com/CodeForAfrica/HURUmap-UI/blob/e5003abcd1c626af975a8d02e54ae0ac660471a8/src/core/InsightContainer/Insight.js#L13-L18
@kilemensi I hope this will be sorted too with the fix you're planning to do
Screenshot from 2019-10-18 16-01-47

**Comment by karimkawambwa** [comment]
@kilemensi @KhadijaMahanga

I resolve insight container issues here https://github.com/CodeForAfrica/HURUmap-UI/pull/67

**Pull request merged**